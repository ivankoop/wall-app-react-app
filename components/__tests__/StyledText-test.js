import React from "react";
import renderer from "react-test-renderer";
import {MonoText} from "../StyledText";

describe("MonoText", () => {
    beforeEach(() => {
        console.error = jest.fn();
    });

    it("renders MonoText", () => {
        jest.useFakeTimers();
        const tree = renderer.create(<MonoText/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
