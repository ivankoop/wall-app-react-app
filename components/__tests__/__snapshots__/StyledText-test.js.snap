// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MonoText renders MonoText 1`] = `
<Text
  style={
    Array [
      undefined,
      Object {
        "fontFamily": "space-mono",
      },
    ]
  }
/>
`;
